OpenOversight/app/templates/description_detail.html

```diff
@@ -1,12 +1,11 @@
 {% extends "base.html" %}
-{% block page_title %}
-  Description Details
-{% endblock page_title %}
-{% block form %}
-  <p>
-    For officer with OOID {{ form.officer_id.data }}.
-    <br>
-    {{ form.description }}
-  </p>
-  {{ super() }}
-{% endblock form %}
+{% block content %}
+  <div class="container theme-showcase" role="main">
+    <div class="page-header">
+      <h1>Viewing Description {{ obj.id }} on Officer {{ obj.officer_id }}</h1>
+    </div>
```

Comment:
Actually, do these changes sync up with the OO branch? I presumed they did, but it looks like a big deviation.

Reply:
@michplunkett  This page wasn't rendering before because the `base.html` page doesn't have `page_title` and `form` blocks. I tested them on LPL OO to make sure they rendered correctly

![1](https://github.com/lucyparsons/OpenOversight/assets/66500457/106dfa69-c168-419f-9b73-425c05588264)
![2](https://github.com/lucyparsons/OpenOversight/assets/66500457/d6a382e1-f38d-40f2-b491-f5ca7a00060e)

Reply:
Here's an example of the current view: https://openoversight.com/officers/83839/notes/6

Reply:
Well done! Looks good to go!